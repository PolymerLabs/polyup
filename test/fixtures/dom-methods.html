<script>
  document.querySelector('input');
</script>

<script>
  // TODO(rictic): make this more robust, test more of the elements.
  Polymer('uses-dom-methods', {
    ready: function() {
      console.log(this.$.child.querySelector('span'));
    }
  });
</script>
