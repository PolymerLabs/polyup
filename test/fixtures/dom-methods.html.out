<script>
  Polymer.dom(document).querySelector('input');
</script>

<script>
  // TODO(rictic): make this more robust, test more of the elements.
  Polymer({
    is: 'uses-dom-methods',
    ready: function () {
      console.log(Polymer.dom(this.$.child).querySelector('span'));
    }
  });
</script>
